tests/src/test/resources/day-time-interval-to-be-fix.csv

```diff
@@ -0,0 +1,2 @@
+interval '106751992' day,,,,interval '2562047789' hour,,,interval '153722867281' minute,,
```

Comment:
May I know what this `to be fix` mean ?

Reply:
Refer to the last test case in CsvScanForIntervalSuite.
A Spark issue blocked this test case: https://issues.apache.org/jira/browse/SPARK-38520